<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jishi</class>
 <widget class="QWidget" name="jishi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1387</width>
    <height>872</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1387</width>
    <height>872</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1387</width>
    <height>872</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="lba">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>250</y>
     <width>331</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.263682 rgba(254, 255, 169, 145), stop:0.37931 rgba(255, 255, 79, 69), stop:0.45 rgba(252, 255, 136, 208), stop:0.477581 rgba(255, 248, 131, 130), stop:0.512315 rgba(255, 233, 142, 130), stop:0.557214 rgba(255, 255, 136, 255), stop:0.57754 rgba(255, 235, 156, 130), stop:0.625 rgba(255, 255, 107, 69), stop:1 rgba(255, 255, 0, 69));</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wenjian">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>90</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resourse/yellow.jfif);
color: rgb(170, 0, 0);
font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="text">
    <string>文件压缩</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>931</width>
     <height>841</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPlainTextEdit" name="shuru">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>621</width>
       <height>331</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.263682 rgba(254, 255, 169, 145), stop:0.37931 rgba(255, 255, 79, 69), stop:0.45 rgba(252, 255, 136, 208), stop:0.477581 rgba(255, 248, 131, 130), stop:0.512315 rgba(255, 233, 142, 130), stop:0.557214 rgba(255, 255, 136, 255), stop:0.57754 rgba(255, 235, 156, 130), stop:0.625 rgba(255, 255, 107, 69), stop:1 rgba(255, 255, 0, 69));</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="shuchu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>621</width>
       <height>331</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.263682 rgba(254, 255, 169, 145), stop:0.37931 rgba(255, 255, 79, 69), stop:0.45 rgba(252, 255, 136, 208), stop:0.477581 rgba(255, 248, 131, 130), stop:0.512315 rgba(255, 233, 142, 130), stop:0.557214 rgba(255, 255, 136, 255), stop:0.57754 rgba(255, 235, 156, 130), stop:0.625 rgba(255, 255, 107, 69), stop:1 rgba(255, 255, 0, 69));</string>
     </property>
    </widget>
    <widget class="QPushButton" name="putf">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>80</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resourse/yellow.jfif);
color: rgb(170, 0, 0);
font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
     </property>
     <property name="text">
      <string>导入文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>200</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resourse/yellow.jfif);
color: rgb(170, 0, 0);
font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
     </property>
     <property name="text">
      <string>清空输入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="suiji">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>80</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resourse/yellow.jfif);
color: rgb(170, 0, 0);
font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
     </property>
     <property name="text">
      <string>随机输入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="copy">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>200</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resourse/yellow.jfif);
color: rgb(170, 0, 0);
font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
     </property>
     <property name="text">
      <string>复制编码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zhuanhuan">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>350</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resourse/yellow.jfif);
color: rgb(170, 0, 0);
font: 12pt &quot;Arial Rounded MT Bold&quot;;
font: 18pt &quot;04b_21&quot;;</string>
     </property>
     <property name="text">
      <string>编码！！！</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>0</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(170, 0, 0);
font: 20pt &quot;Arial Rounded MT Bold&quot;;

background-image: url(:/resourse/biankuang.png);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#00aaff;&quot;&gt;编码模式&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>51</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(170, 0, 0);
font: 14pt &quot;Arial Rounded MT Bold&quot;;

background-image: url(:/resourse/biankuang.png);</string>
     </property>
     <property name="text">
      <string>编码信息</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextEditable</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>0</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPlainTextEdit" name="shuru_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>621</width>
       <height>331</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.263682 rgba(254, 255, 169, 145), stop:0.37931 rgba(255, 255, 79, 69), stop:0.45 rgba(252, 255, 136, 208), stop:0.477581 rgba(255, 248, 131, 130), stop:0.512315 rgba(255, 233, 142, 130), stop:0.557214 rgba(255, 255, 136, 255), stop:0.57754 rgba(255, 235, 156, 130), stop:0.625 rgba(255, 255, 107, 69), stop:1 rgba(255, 255, 0, 69));</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="shuchu_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>621</width>
       <height>331</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.263682 rgba(254, 255, 169, 145), stop:0.37931 rgba(255, 255, 79, 69), stop:0.45 rgba(252, 255, 136, 208), stop:0.477581 rgba(255, 248, 131, 130), stop:0.512315 rgba(255, 233, 142, 130), stop:0.557214 rgba(255, 255, 136, 255), stop:0.57754 rgba(255, 235, 156, 130), stop:0.625 rgba(255, 255, 107, 69), stop:1 rgba(255, 255, 0, 69));</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zhuanhuan2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>260</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resourse/yellow.jfif);
color: rgb(170, 0, 0);
font: 12pt &quot;Arial Rounded MT Bold&quot;;
font: 18pt &quot;04b_21&quot;;</string>
     </property>
     <property name="text">
      <string>译码！！！</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>130</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resourse/yellow.jfif);
color: rgb(170, 0, 0);
font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
     </property>
     <property name="text">
      <string>清空输入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="copy2">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>130</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resourse/yellow.jfif);
color: rgb(170, 0, 0);
font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
     </property>
     <property name="text">
      <string>复制文本</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>0</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(170, 0, 0);
font: 20pt &quot;Arial Rounded MT Bold&quot;;

background-image: url(:/resourse/biankuang.png);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff5500;&quot;&gt;译码模式&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(170, 0, 0);
font: 14pt &quot;Arial Rounded MT Bold&quot;;

background-image: url(:/resourse/biankuang.png);</string>
     </property>
     <property name="text">
      <string> 译码信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>0</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="change">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>90</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resourse/yellow.jfif);
color: rgb(170, 0, 0);
font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="text">
    <string>模式切换</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>170</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;Arial Rounded MT Bold&quot;;
background-image: url(:/resourse/biankuang.png);</string>
   </property>
   <property name="text">
    <string>字符编码列表</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="search2">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>810</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.263682 rgba(254, 255, 169, 145), stop:0.375 rgba(255, 255, 0, 69), stop:0.45 rgba(252, 255, 136, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.557214 rgba(255, 255, 136, 255), stop:0.57754 rgba(255, 221, 85, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="search">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>750</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.263682 rgba(254, 255, 169, 145), stop:0.375 rgba(255, 255, 0, 69), stop:0.45 rgba(252, 255, 136, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.557214 rgba(255, 255, 136, 255), stop:0.57754 rgba(255, 221, 85, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>730</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 0, 0);
font: 14pt &quot;Arial Rounded MT Bold&quot;;
background-image: url(:/resourse/biankuang.png);
</string>
   </property>
   <property name="text">
    <string>字符编码查询</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lba</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>wenjian</zorder>
  <zorder>change</zorder>
  <zorder>label_3</zorder>
  <zorder>search2</zorder>
  <zorder>search</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
